libs/pando/ui-sheet/src/field.ts

```diff
@@ -0,0 +1,3 @@
+export type Field = {
+  fieldRef: string
```

Comment:
(does not have to be this PR)Need to associate a translation and icon to this field for display.